nuxeo-web-ui-ftest/webdriver/test/features/cloud.feature

```diff
@@ -0,0 +1,45 @@
+@watch
+Feature: Cloud Services
+
+  Cloud providers can be added, edit and removed.
+
+  Background:
+    Given I login as "Administrator"
+
+  Scenario: Cloud Services menu on drawer
+    When I click the "administration" button
+    Then I can see the administration menu
+    When I click "Cloud services" in the administration menu
+    Then I can see the cloud services page
+
+  Scenario: Simple cloud providers view, create, edit, delete
```

Comment:
We should split created, edit and delete into different scenarios. If creating fails, we'll never know whether editing and deleting work or not.